src/features/login/loginSlice.ts

```diff
@@ -0,0 +1,7 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const loginSlice = createSlice({
```

Comment:
This state slice doesn't have an initial state or reducers. I suppose we will need this later?

Reply:
Also, it might be better to name this auth slice but I don't have a problem with it staying this way.